com.github.varnholt.dynablaster_revenge.json

```diff
@@ -0,0 +1,55 @@
+{
+    "app-id": "com.github.varnholt.dynablaster_revenge",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=pulseaudio"
+    ],
+    "command": "dynablaster.sh",
+    "cleanup": [
+        "Makefile"
+    ],
+    "modules": ["shared-modules/glu/glu-9.json",
+    {
+        "name": "dynablaster",
+        "buildsystem": "qmake",
+        "no-make-install": true,
+        "subdir": "client",
+        "sources": [
+            {
+                "type": "git",
+                "url": "https://github.com/varnholt/dynablaster_revenge.git",
+                "commit": "08fb11fb2d5dd94e1ea0e447d95cccb375dfc7e5"
```

Comment:
any reason to not use the tagged release?

Reply:
There were fixes to the project since then, and the project doesn't update often anyways.

Reply:
what are these fixes?

Reply:
Memory leak fixes, some build fixes